spec/controllers/books_controller_spec.rb

```diff
@@ -7,17 +7,16 @@
 
   describe 'GET #index' do
     context 'When fetching all the books' do
-      let!(:books) { create_list(:book, 20) }
+      let(:books) { create_list(:book, 20) }
 
       before do
         get :index
       end
 
       it 'responses with the books json' do
-        expected = ActiveModelSerializers::SerializableResource.new(
+        expect(response_body.to_json) == ActiveModelSerializers::SerializableResource.new(
```

Comment:
use `.to eq` like you did in the other example

Reply:
when I deleted the variable 'expected', I threw this error and the '==' suggestion when running the tests
![image](https://user-images.githubusercontent.com/15805572/40755009-63b7a524-644a-11e8-8f1d-17e5f9de3875.png)